client/tmclient/screens.py

```diff
@@ -209,18 +212,40 @@ def __init__(self, client_state, loop):
 
         self.focus_prompt()
 
+    def launch_witch(self, data):
+        with open("witch.hy", "w") as f:
```

Comment:
we should probably be using [tempfile](https://docs.python.org/3/library/tempfile.html#tempfile-examples)